.editorconfig

```diff
@@ -6,9 +6,9 @@ insert_final_newline = true
 
 [*.java]
 charset = utf-8
-indent_style = tab
+indent_style = space # 4 spaces, remember!
```

Comment:
Out of scope for PR, and especially don't mess with editorconfig since it affects literally the entire project. I assume the "4 spaces" you're referencing are from the GC5 contributing guidelines, don't mix projects like that.